changelog/next/features/3471--pausing-and-resuming-pipelines.md

```diff
@@ -0,0 +1,4 @@
+It is now possible to pause and resume pipelines in the pipeline manager with
+the `pause` and `start` actions.
+
+Newly created pipelines are now in a new `created` state.
```

Comment:
Minor nit: always prefer active voice for documentation and changelog entries.

```suggestion
The `pause` action in the `/pipeline/update` endpoint suspends a pipeline and
sets its state to `paused`. Resume it with the `start` action.

Newly created pipelines are now in a new `created` rather than `stopped` state.
```

Reply:
I changed the changelog entry.